Library/Homebrew/unpack_strategy/tar.rb

```diff
@@ -38,8 +39,9 @@ def extract_to_dir(unpack_dir, basename:, verbose:)
         end
 
         system_command! "tar",
-                        args:    ["xf", tar_path, "-C", unpack_dir],
-                        verbose: verbose
+                        args:         ["xf", tar_path, "-C", unpack_dir],
+                        extra_params: extra_params ||= [],
```

Comment:
Rename this to `extra_args` and use 

```ruby
args: ["xf", tar_path, "-C", unpack_dir, *extra_args],
```

above instead.

Reply:
Ok I am changing now committed squashed and force pushed/